src/table/settings/network_settings.ini

```diff
@@ -66,6 +66,15 @@ min      = 1
 max      = 65535
 cat      = SC_EXPERT
 
+[SDTC_VAR]
+var      = network.commands_per_frame_server
+type     = SLE_UINT16
+flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC | SF_NETWORK_ONLY
+def      = 16
```

Comment:
What's the 16 based on? If a game script  need to manage a few hundred/thousand towns/industries, will that be enough?
1024/16 = 64 => ~1 game day. 

Alternatively, is a 16x improvement over 13.x behaviour enough? Or is that still too limiting?
Should the server maybe not have any limits, but only for game scripts?

Or it 16x a nice middle ground for some performance improvements of game scripts without immediately overwhelming? And is it up to the server owners to (occasionally) set this setting (much) higher for specific game scripts on especially large maps?

Reply:
It's an arbitrary value which is a compromise between allowing a larger throughput of commands from the server and not allowing a misbehaving game script to cause the network/game overall to grind to a halt.

Making it a setting avoids excessive bikeshed problems whilst providing better default behavior than currently.